network/load-balancer/how-to/add-certificate.mdx

```diff
@@ -0,0 +1,162 @@
+---
+meta:
+  title: How to add a certificate to your Load Balancer
+  description: This page explains how to add a certificate to your Load Balancer
+content:
+  h1: How to add a certificate to your Load Balancer
+  paragraph: This page explains how to add a certificate to your Load Balancer
+dates:
+  validation: 2022-10-20
+  posted: 2022-10-20
+---
+
+You can add a certificate to your Load Balancer's frontend to enable the Load Balancer to encrypt/decrypt incoming traffic, necessary for [SSL bridging and SSL offloading configurations](/network/load-balancer/reference-content/configuring-load-balancer).
+
+<Message type="tip">
+
+Originally, certificates used the **S**ecure **S**ockets **L**ayer (SSL) cryptographic protocol, and are still often referred to simply as SSL certificates. However, it is now standard for such certificates not to use SSL at all, but rather its successor: **T**ransport **L**ayer **S**ecurity (TLS). [Transport Layer Security](https://en.wikipedia.org/wiki/Transport_Layer_Security) is a more up to date security protocol designed to increase privacy and data security for communications over the Internet. 
+
+You may therefore hear certificates referred to as "SSL certificates", "TLS certificates" or "SSL/TLS certificates". These are essentially all the same thing.
+
+</Message>
+
+When your Load Balancer's frontend has a certificate, clients can securely connect to it via an encrypted connection. The underlying process is as follows:
+
+- The client attempts to securely connect to the Load Balancer and asks it to identify itself.
+- The Load Balancer sends the client a copy of its certificate in response.
+- The client attempts to verify the certificate, and signals to the Load Balancer if it is satisfied it can be trusted.
+- The Load Balancer returns an acknowledgment to start an encrypted session.
+- Encrypted data is sent to the Load Balancer, who can then decrypt it using its certificate.
```

Comment:
Be careful here the certificate is not used to decrypt traffic. It is only used to establish the secured connection with the client. It's the key and algorithm defined during the TLS/SSL handshake that are used to decrypt the traffic.